dist/pom.xml

```diff
@@ -450,6 +452,31 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-install-plugin</artifactId>
+                <version>3.0.1</version>
+                <executions>
+                    <execution>
+                        <id>default-install</id>
+                        <phase>none</phase>
+                    </execution>
+                    <execution>
+                        <id>install-parallel-worlds-jar</id>
+                        <phase>install</phase>
+                        <goals>
+                            <goal>install-file</goal>
+                        </goals>
+                        <configuration>
+                            <version>${project.version}</version>
```

Comment:
In maven central, we deployed pom.xml is : `dist/target/extra-resources/META-INF/maven/com.nvidia/rapids-4-spark_2.12/pom.xml`  https://github.com/NVIDIA/spark-rapids/blob/branch-22.10/jenkins/deploy.sh#L59

Here we generated a new  pom.xml generated by mvn CLI "install:install-file"  instead, which is also diff to the pom in maven central. 

Could you help to explain: how does it fix the issue ? :  https://github.com/NVIDIA/spark-rapids/issues/6433 [dist pom file need to be identical, no matter it is installed locally, or pulled from maven central ]


```
**Expected behavior**
The pom in local m2 should be identical[ the same to maven central?] if there are no changes made locally
```

Reply:
@NvTimLiu  with this PR we are making sure that calling `mvn install` locally
and `mvn deploy:deploy-file` in the script use the same reduce pom. Currently install is using the original  pom.

We probably can do the same for "mvn deploy" such that we don't need to call deploy-file manually. I'll update the PR

Reply:
@gerashegalov  Thanks a lot, your explanation solved my question!

I ran `mvn install:install-file xxx` CLI locally, and found it created a new pom.xml (if I did not specifying the -Dpomfile=xxx) instead of reusing `extra-resources/META-INF/maven/xxx/pom.xml` file in the dist jar, 

suppose this is MVN CLI issue. 

mvn install:install `maven-install-plugin`  works well, it references to the right `extra-resources/META-INF/maven/xxx/pom.xml` file in the dist jar.